vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -235,6 +238,7 @@ type internal FSharpQuickInfoProvider
                           | None, None -> ()
                           | None, Some _ -> yield! sigDocumentation
                           | Some _, None -> yield! targetDocumentation
+                          // TODO: handle case when one is substring of the other
```

Comment:
In another PR. Probably opening an issue would be better than putting a comment there, I'll remove it.